cli/flox/src/commands/generations/list.rs

```diff
@@ -44,33 +47,52 @@ struct DisplayMetadata<'m> {
 }
 impl Display for DisplayMetadata<'_> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        writeln!(f, "Description: {}", self.metadata.description)?;
-        write!(f, "Created: {}", self.metadata.created)?;
-        if let Some(last_active) = self.metadata.last_active {
-            writeln!(f)?;
-            write!(f, "Last Active: {last_active}")?;
+        let description = &self.metadata.description;
+        let created = self.metadata.created;
+        let last_active = if let Some(last_active) = self.metadata.last_active {
+            last_active.to_string()
+        } else {
+            "Now".to_string()
         };
-        Ok(())
+
+        write!(f, "{}", formatdoc! {"
+            Description: {description}
+            Created:     {created}
+            Last Active: {last_active}"})
     }
 }
 
 /// Formatter container for [AllGenerationsMetadata].
 /// List formatting of generation data, following the template
 ///
+/// Current version:
 /// ```text
-/// * <generation id>[ (current)]:
+/// * Generation: <generation id> (current)
 ///   <generation metadata>          # implemented by [DisplayMetadata] above
 /// ```
+///
+/// Other versions:
+/// ```text
+///   Generation: <generation id>
+///   <generation metadata>
+/// ```
 struct DisplayAllMetadata<'m>(&'m AllGenerationsMetadata);
 impl Display for DisplayAllMetadata<'_> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let mut iter = self.0.generations().into_iter().peekable();
         while let (Some((id, metadata)), peek) = (iter.next(), iter.peek()) {
-            write!(f, "* {id}")?;
-            if Some(id) == self.0.current_gen() {
-                write!(f, " (current)")?;
+            let generation = format!("Generation:  {id}");
+            let (line, current) = if Some(id) == self.0.current_gen() {
+                (format!("* {generation}"), " (current)")
```

Comment:
Nit: the asterisk and `(current)` feel redundant, and we may want to make the text follow our color scheme of purple instead of yellow